src/rustdoc/proposals-and-stabilization.md

```diff
@@ -0,0 +1,75 @@
+# Proposals, Approvals and Stabilization
+It is very common to need to gather feedback and approval when contributing to rustdoc, either
+for permission to proceed with an experiment or refactoring, or when stabilizing a feature. This
+document aims to summarise the various processes that the rustdoc team has for making approval
+decisions and when each should be used.
+
+## Approvals
+There are two mechanisms that the team can use to approve a proposal (not all approval mechanisms
+are suitable for each method of making a proposal - see below):
+
+- r+
+  - A proposal is r+'d when it is approved to be merged.
+  - r+ can only be used to approve a PR.
+- FCP
+  - A final comment period will require sign-off from a majority of the rustdoc team to approve
+    a proposal and then a ten day waiting period.
+  - FCPs can be used to approve any form of proposal.
+
+## Proposals
+There are three ways to propose a change to the rustdoc team. The appropriate choice depends on
+the nature of the proposal, described below.
+
+- Open a discussion on the [rustdoc zulip thread].
+  - This is the preferred way. It allows to prevent users to lose too much time implementing
+    something if in the end, the team will ask major changes or even refuse it. After the
+    discussion, if accepted and depending on the change, an RFC or a PR will be the next step.
+- Request For Comments (RFC)
+  - RFCs are pull requests to the [`rust-lang/rfcs`][rfcs] repository and are a heavy-weight
+    proposal mechanism, reserved for significant changes.
+  - RFC proposals can only be approved by *FCPs*.
+- Pull Request (PR)
+  - PRs are pull requests to the [`rust-lang/rust`][rust] repository and are a light-weight
+    proposal mechanism, suitable for most proposals. PRs are preferred when the proposal is
+    accompanied by a small patchset (such as stabilization of a compiler flag or addition of
+    a new target).
+  - PR proposals can be approved by *FCPs* or *r+*.
```

Comment:
Nit: maybe also spell out "see *When are FCPs/RFCs required* for when r+ isn't sufficient alone"